spanner/tests/unit/test_snapshot.py

```diff
@@ -252,9 +252,9 @@ def _read_helper(self, multi_use, first=True, count=0):
         else:
             self.assertIsNone(result_set._source)
 
-        result_set._consume_all()
+#        result_set._consume_all()
```

Comment:
Please don't leave commented-out code checked in.

Reply:
oops.  fixed.